### PR TITLE
[message] ingester request add org field

### PR DESCRIPTION
**Pull request by lzf575**
### This PR is for:

- Server

### CODE REVIEW

File: message/trident.proto [hunk 1]
```diff
@@ -115,6 +116,8 @@ message SyncRequest {
 
     optional string kubernetes_cluster_id = 45;    // 仅对容器类型的采集器有意义
     optional string kubernetes_cluster_name = 46;  // 仅对容器类型的采集器有意义
+
+    optional uint32 org_id = 50;  // only valid when requested by Analyzer
```
Comment (sharang) [inline message/trident.proto:120]: default 1

Comment (sharang) [reply message/trident.proto:120]: // only used by ingester

File: message/trident.proto [hunk 2]
```diff
@@ -1110,3 +1115,9 @@ message RemoteExecResponse {
     repeated LinuxNamespace linux_namespaces = 3;
     optional CommandResult command_result = 4;
 }
+
+message OrgIDsRequest {}
+
+message OrgIDsResponse {
+    repeated uint32 org_id = 1;
```
Comment (sharang) [inline message/trident.proto:1122]: org_ids

**Review by jin-xiaofeng — DISMISSED** [review]

File: message/trident.proto (hunk 2 above)
Comment (lzf575) [reply message/trident.proto]: ok

File: message/trident.proto (hunk 1 above)
Comment (lzf575) [reply message/trident.proto]: ok

**Pull request merged**